src/analytics.js

```diff
@@ -0,0 +1,15 @@
+const SegmentAnalytics = require('analytics-node')
+
+class Analytics {
+  constructor (writeKey) {
+    this.client = writeKey ? new SegmentAnalytics(writeKey) : {}
+  }
+
+  track (payload) {
```

Comment:
This is not really what I meant. Sorry if I was unclear. Now you are only wrapping the segment class. I meant that this class would have multiple functions for the different event's. So one method for each of the following: `infra_metrics`, get_profile, open_box, and PinData.

Reply:
In the pinning class we would then only need to call the specific methods, no need to do payload.